### ISSUE TITLE
Design document for implementing multiclass skills

### ISSUE THREAD

**Issue by wellingtj** [uchicago-cs/chiventure#1550]
Some skills might need more than one class to activate, and so I thought it might be worth coming up with a way to create skills that need multiple classes, called multiclass skills. This issue seeks to come up with a way to address this. Not only that, but multiclassing needs to be addressed in the wiki now that the multiclass module is updated. I plan to address the basic idea of what a multiclass is, and how the skills/stats/attributes change when there is a multiclass as well.

**Comment by wellingtj** [comment]
The design document is complete. I will upload it to the wiki soon and then close this issue

**Comment by wellingtj** [comment]
the wiki has been updated accordingly and the design document is completed. See the Multiclass: Design section of the wiki to get a bit more of an idea for functionality. See issue #1568 for implementation of this issue. Here is the link to the page : https://github.com/uchicago-cs/chiventure/wiki/Player-Class-~-Multiclassing:-Design. Within here, I wrote a brief summary detailing what a multiclasss actually is, the key features of a multiclass, and the implementation (how to combine two classes with skills, stats, and attributes).
